pkg/compatibility/compatibility.go

```diff
@@ -49,6 +50,7 @@ type VersionInfo struct {
 }
 
 var (
+	ocpBetaVersions   = []string{"4.13", "4.14"}
```

Comment:
I guess this variable would be updated then these versions beceome GA (or when a new beta OCP version is released), wouldn't it?

Reply:
Yeah, unfortunately it's a manual process like the `ocpLifeCycleDates` variable.  Luckily the values don't change often.